var/spack/repos/builtin/packages/tasmanian/package.py

```diff
@@ -153,10 +157,54 @@ def cmake_args(self):
 
         return args
 
+    # TODO: Replace this method and its 'get' use for cmake path with
+    #   join_path(self.spec['cmake'].prefix.bin, 'cmake') once stand-alone
+    #   tests can access build dependencies through self.spec['cmake'].
+    def cmake_bin(self, set=True):
+        """(Hack) Set/get cmake dependency path."""
+        filepath = join_path(self.install_test_root, 'cmake_bin_path.txt')
+        if set:
+            with open(filepath, 'w') as out_file:
+                cmake_bin = join_path(self.spec['cmake'].prefix.bin, 'cmake')
+                out_file.write('{0}\n'.format(cmake_bin))
+        elif os.path.isfile(filepath):
+            with open(filepath, 'r') as in_file:
+                return in_file.read().strip()
+
+    @run_after('install')
+    def setup_smoke_test(self):
+        if not self.spec['cmake'].satisfies('@3.22:'):
```

Comment:
Tasmanian requires CMake 3.10 or newer. The test requires the same CMake version used in the build, or newer. If you build Tasmanian with CMake 3.10, then you can test with CMake 3.10. If you build with CMake 3.22, then you need CMake 3.22.